specification/cognitiveservices/data-plane/Language/stable/2022-05-01/conversationalanalysisauthoring.json

```diff
@@ -0,0 +1,3060 @@
+          "Successful Get Supported Prebuilt Entities": {
+            "$ref": "./examples/conversationalanalysisauthoring/SuccessfulGetSupportedPrebuiltEntities.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink",
+          "itemName": "value"
+        }
+      }
+    },
+    "/authoring/analyze-conversations/projects/global/training-config-versions": {
+      "get": {
+        "description": "Lists the support training config version for a given project type.",
+        "operationId": "ConversationalAnalysisAuthoring_ListTrainingConfigVersions",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ConversationalAnalysisAuthoringProjectKindInQuery"
+          },
+          {
+            "$ref": "common.json#/parameters/TopParameter"
+          },
+          {
+            "$ref": "common.json#/parameters/SkipParameter"
+          },
+          {
+            "$ref": "common.json#/parameters/MaxPageSizeParameter"
+          },
+          {
+            "$ref": "common.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "List of config versions.",
+            "schema": {
+              "$ref": "#/definitions/ConversationalAnalysisAuthoringTrainingConfigVersions"
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "common.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Successful Get Supported Training Config Versions": {
+            "$ref": "./examples/conversationalanalysisauthoring/SuccessfulGetSupportedTrainingConfigVersions.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink",
+          "itemName": "value"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ConversationalAnalysisAuthoringCompositionSetting": {
+      "enum": [
+        "returnLongestOverlap",
+        "requireExactOverlap",
+        "separateComponents",
+        "combineComponents"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "CompositionSetting",
+        "modelAsString": true
+      },
+      "x-ms-client-name": "CompositionSetting"
+    },
+    "ConversationalAnalysisAuthoringConfusionMatrix": {
+      "type": "object",
+      "additionalProperties": {
+        "$ref": "#/definitions/ConversationalAnalysisAuthoringConfusionMatrixRow"
+      },
+      "x-ms-client-name": "ConfusionMatrix"
+    },
+    "ConversationalAnalysisAuthoringConfusionMatrixEntry": {
+      "required": [
+        "normalizedValue",
+        "rawValue"
+      ],
+      "type": "object",
+      "properties": {
+        "normalizedValue": {
+          "format": "float",
+          "description": "Represents normalized value in percentages.",
```

Comment:
I'm unclear how this relates to `rawValue`, if at all. Is this just `rawValue` * 100? If not, throughout Language we've been representing percentages as 0.0 to 1.0 and you should specify the range. This seems more common in swaggers within and without Microsoft.

Reply:
`rawValue` represents the value that created the percentages. For example, the number of documents that were correctly predicted is 3 out of 4, then `rawValue` would be 3 and `normalizedValue` would be 75.

Reply:
Since this doesn't represent a score, and we also use percentage to be 0-100 in the JobState to represent the progress, I would prefer to keep the values 0-100 rather than 0-1.